woudc_data_registry/search.py

```diff
@@ -374,7 +374,8 @@ def create(self):
             if not definition['enabled']:
                 continue
 
-            index_name = definition['index']
+            index_name = '{}.{}'.format(config.WDR_SEARCH_INDEX_BASENAME,
```

Comment:
If config.WDR_SEARCH_INDEX_BASENAME is empty, do we want to keep the period (which would then be at the front of the string)?